docs/apis/subsystems/form/index.md

```diff
@@ -300,6 +300,29 @@ public function definition() {
 }
 ```
 
+### filter_shown_headers()
+
+<Since version="4.3" />
+
+<!-- cspell:ignore _shownonlyelements -->
+This method adds values to _shownonlyelements array to decide whether a header should be shown or hidden.
```

Comment:
The description of this method should also mention that:

- It only applies to header elements.
- When it's used, the elements will be displayed and expanded and the rest of the headers will be hidden.

Reply:
I was expecting also some example/mention to say that a couple of forms, editsections and modedit, support it currently (as a sample to show how it works).

Reply:
> The description of this method should also mention that:
> 
> * It only applies to header elements.
> * When it's used, the elements will be displayed and expanded and the rest of the headers will be hidden.

I'll try to explain that it only applies to header elements in the description saying "whether a heder", but I guess it was not enough :-)

I made a new proposal. 
I've also added some life example following same approach we are following in disable_form_change_checker(). I think this makes more sense and gives more context than adding a piece of isolated code only, but we have a problem because those links will be incorrect till the issue is integrated, and can be wrong in the future too. So let me know if you think I should a different approach to the one already exists.